packages/i18n/src/locales/en.i18n.json

```diff
@@ -6265,7 +6267,7 @@
   "Always_send_the_transcript_to_contacts_at_the_end_of_the_conversations": "Always send the transcript to contacts at the end of the conversations.",
   "Export_conversation_transcript_as_PDF": "Export conversation transcript as PDF",
   "Omnichannel_transcript_email": "Send chat transcript via email.",
-  "Accounts_Default_User_Preferences_omnichannelTranscriptEmail_Description": "Always send the transcript to contacts at the end of the conversations.",
+  "Accounts_Default_User_Preferences_omnichannelTranscriptEmail_Description": "Always send the transcript to contacts at the end of the conversations. This preference may be overriden by an admin setting.",
```

Comment:
I believe we should delete all other translations that will be outdated with this change so that they're re-translated later (lingohub doesn't update lines, it just inserts new ones).

Reply:
Yikes. How can we confirm that? Don't wanna remove all the other translations without eing sure they won't get updated otherwise 👀